packages/babel-parser/src/tokenizer/state.js

```diff
@@ -33,6 +33,12 @@ export default class State {
     this.hasFlowComment = false;
     this.isIterator = false;
 
+    // Used by smartPipelines.
+    this.topicContextState = {
```

Comment:
Nit: this can be renamed to "topicContext", since "State" is already part of the class name.